scripts/lint_prowjobs.go

```diff
@@ -0,0 +1,279 @@
+/*
+Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
+
+	prowapi "k8s.io/test-infra/prow/config"
+	yaml "sigs.k8s.io/yaml"
+)
+
+type PresubmitConstants struct {
+	Bucket                   string
+	Cluster                  string
+	ServiceAccountName       string
+	DefaultMakeTarget        string
+	CniMakeTarget            string
+	HelmMakeTarget           string
+	ReleaseToolingMakeTarget string
+}
+
+type PostsubmitConstants struct {
+	Bucket                       string
+	Cluster                      string
+	DefaultServiceAccountName    string
+	AthensHelmServiceAccountName string
```

Comment:
This will require us to add any new service account used in postsubmits to this script, can we just drop the service account check from postsubmits?